presto-docs/src/main/sphinx/include/PrestoThriftService.thrift

```diff
@@ -209,14 +214,18 @@ struct PrestoThriftSplit {
   2:  list<PrestoThriftHostAddress> hosts;
 }
 
-struct PrestoThriftPageResult {
+struct PrestoThriftPage {
   /**
    * Returns data in a columnar format.
    * Columns in this list must be in the order they were requested by the engine.
    */
   1:  list<PrestoThriftBlock> columnBlocks;
   2:  i32 rowCount;
-  3: optional PrestoThriftId nextToken;
+}
+
+struct PrestoThriftPageResult {
+  1:  PrestoThriftPage page;
```

Comment:
I'm concerned about this not being backwards compatible. It would require a staggered deployment for existing clients to try to migrate to the new version. The only way I can think for them doing it without downtime would be that they'd need to spin up a new version of their thrift service, update to the new version of presto pointing to that new service, then shut down the old version. 

I'm curious how others feel.

Reply:
I discussed it with Dain and we decided to not introduce the new `PrestoThriftPage` class. I'm going to remove it from the PR.